Tests/Web/html/tests/logsystem_tests.js

```diff
@@ -0,0 +1,58 @@
+import { test, assert } from '../test_framework.js';
+
+import { Systems } from '../connected_spaces_platform.js';
+
+function logCallback(message) {
```

Comment:
This function is never used

Reply:
Ah yeah, I changed to the below way of doing it to include the `messageLogged` boolean. Will remove